addons/MountRoulette/MountRoulette.lua

```diff
@@ -86,4 +102,73 @@ windower.register_event('addon command', function()
     -- Generate random number and use it to choose a mount
     local mount_index = math.ceil(math.random() * #allowed_mounts)
     windower.send_command('input /mount "' .. allowed_mounts[mount_index] .. '"')
+end
+
+commands.blacklist = function(args)
+    local operation = args:remove(1)
+
+    if not operation then
```

Comment:
Am I reading this right if you don't supply any arguments and just do `//mr` or `//mr mount` it would just list the blacklisted mounts?

Reply:
If you do `//mr`, line 162 defaults the command to `mount`. If you define the command, like `//mr mount` or `//mr blacklist`, the correct command is referenced from the `commands` table.

Reply:
Yeah, but removing if you don't provide a further option won't `operation` be `nil` here?

Reply:
Oh yeah, if you do `//mr blacklist` it just prints all blacklisted mounts, then returns.

Reply:
Oh nevermind me, I just realized what's happening here. I was misreading how the command handler works... sorry about that.